### ISSUE TITLE
couldn't know some files' source

### ISSUE THREAD

**Issue by zimeizhng** [lishuya17/PocketAnchorData#5]
I'm sorry, i looked through all your code( including PocketAnchor), but I still don't know how to generate these files. i would be really greatful if you can help me!
![image](https://github.com/lishuya17/PocketAnchorData/assets/71952012/8157ea95-63c3-4162-a566-bdda22b8dd35)

**Comment by lishuya17** [comment]
Since it was not convenient to upload all the data files on GitHub, we put them in the docker image (https://hub.docker.com/r/tiantz17/pocketanchor).

Sorry for the inconvenience it caused!

**Comment by zimeizhng** [comment]
no, no, no, i can find these files, but i don't know how to generate these files and what they mean!

**Comment by lishuya17** [comment]
I found my code for processing molecular features:

[molecule_data_process.zip](https://github.com/lishuya17/PocketAnchorData/files/13526185/molecule_data_process.zip)

Hope this can help you. Please contact me if you have further questions.
